pkg/network/usm/monitor_test.go

```diff
@@ -617,7 +617,8 @@ func (s *USMHTTP2Suite) TestHTTP2ManyDifferentPaths() {
 	defer monitor.Stop()
 
 	// Should be bigger than the length of the http2_dynamic_table which is 1024
-	numberOfRequests := 1500
+	const numberOfRequests = 1500
+	const expectedNumberOfRequests = numberOfRequests * 2
```

Comment:
maybe add a const which will be better explain the issue, something like `expectedNumberOfRequestsWithRST`